sdk/core/System.ClientModel/README.md

```diff
@@ -21,15 +23,21 @@ dotnet add package System.ClientModel
 
 None needed for `System.ClientModel`.
 
+### Authenticate the client
+
+The `System.ClientModel` preview package provides a `KeyCredential` type for authentication.
```

Comment:
The use of "preview" here seems like something that we could easily overlook and forget about.  Should we remove it?

Reply:
Will fix.